crates/sui-types/src/sui_system_state.rs

```diff
@@ -277,6 +283,63 @@ impl SuiSystemState {
         }
     }
 
+    pub fn get_current_epoch_peer_ids(&self) -> HashMap<PeerId, String> {
```

Comment:
I am not 100% sure this method gives additional value in the SUI validator case, as a validator will just try to connect to other validators (we don't have the same distinction between primary/workers here). Could you please provide a bit more info on what's the intention here?